models/test/mapper/TestPreviewMapper.php

```diff
@@ -230,11 +231,16 @@ private function getService(): Service
 
     /**
      * @param $categories
-     *
+     * @param $itemRef
      * @return bool
      */
-    private function isItemInformational($categories): bool
+    private function isItemInformational($categories, $itemRef): bool
     {
-        return in_array('x-tao-itemusage-informational', $categories, true);
+        $additionalCheck = false;
```

Comment:
do we need to modify tests ?

Reply:
\oat\taoQtiTestPreviewer\test\unit\models\test\factory\TestPreviewMapperTest maybe?

Reply:
You are introducing parameter that is not sharing common interface and you are not declaring type in parameter. This seems sketchy.

Reply:
@bartlomiejmarszal 
Well, I see all the code is relying on` AssesmentItemRef` but actually https://github.com/oat-sa/qti-sdk/blob/master/src/qtism/runtime/tests/RouteItem.php#L154 in spite of phpDoc returns `ExtendedAssessmentItemRef` class object and it gives us an ability to check does the item have response declarations. I can not change/set type declarations because as I understand it depends on QTI content version and can brake compatibility, but in practice it solves this particular issue (testPreview do not automatically mark items informational although testRunner do) so I think we can go with it

Reply:
ExtendedAssessmentItemRef actually extends AssessmentItemRef.
It seems that for practical reasons, ExtendedAssessmentItemRef's are stored in test session instead of AssessmentItemRef's so I think you can safely rely on it. I'll include this phpdoc change in a future release.